jobs/JGLOBAL_PREP_SNOW_OBS

```diff
@@ -12,17 +24,19 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "prepsnowobs" -c "base prepsnowobs"
 GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
 gPDY=${GDATE:0:8}
 gcyc=${GDATE:8:2}
-GDUMP="gdas"
 
 ##############################################
 # Begin JOB SPECIFIC work
 ##############################################
 # Generate COM variables from templates
-YMD=${PDY} HH=${cyc} generate_com -rx COM_OBS
+RUN=${oCDUMP} MEMDIR="" YMD=${PDY} HH=${cyc} generate_com -rx COM_OBS_IN:COM_OBS_TMPL
```

Comment:
There are a couple things I don't like here, the main one being this will not be acceptable to operations where `COM_OBS_TMPL` points to the obsproc space and should be strictly input for GFS. The out needs to use a different template (probably a new `COM_SNOW_PREP_TMPL`).

Also, if IN and OUT are going to be used in the variable name, would prefer to have it be `COMIN_*` and `COMOUT_*`. That is the pattern we plan on using when we add IN and OUT to the rest of the workflow.

Reply:
Thanks for the good suggestions, and I have made changes from `COM_OBS_IN/OUT` to `COMIN_OBS` and `COMOUT_OBS`. I have no idea and wait for the decisions on using a new `COM_SNOW_PREP_TMPL` or not.